src/components/ExtensionHub.ts

```diff
@@ -35,6 +36,11 @@ const FEATURE_OPTIONS: FeatureOption[] = [
     label: 'Chat',
     component: () => html`<moz-chat></moz-chat>`,
   },
+  {
+    value: 'planner',
+    label: 'Planner',
+    component: () => html`<moz-planner></moz-planner>`,
```

Comment:
ha, i named the same value and component in https://github.com/Firefox-AI/extension-hub/pull/11 but i'll rename mine to moz-plan-checklist @nickgrato not sure if we have a cleaner way to avoid merge conflicts but at least simple to resolve